tests/src/Glpi/Tests/CommonDBTestCase.php

```diff
@@ -188,7 +188,8 @@ public function checkInstall($filename = '', $filter = 'glpi_', $when = '') {
                $s_type[0] = str_replace(" COLLATE utf8_unicode_ci", "", $s_type[0]);
                $s_type[0] = str_replace(" CHARACTER SET utf8", "", $s_type[0]);
                if (strpos(trim($s_type[0]), 'text') === 0
-                   || strpos(trim($s_type[0]), 'longtext') === 0) {
+                  || strpos(trim($s_type[0]), 'mediumtext') === 0
+                  || strpos(trim($s_type[0]), 'longtext') === 0) {
```

Comment:
i thought I disabled the call to this method long ago.

GLPI provides tools to compare an upgraded DB against a fresh install DB. This is more useful than this stuf, which only checks if the empty sql file was correctly processed.

Please re-move this method and its call. This is obsolete.